activesupport/lib/active_support/log_subscriber.rb

```diff
@@ -29,13 +29,36 @@ module ActiveSupport
   # subscriber, the line above should be called after your
   # <tt>ActiveRecord::LogSubscriber</tt> definition.
   #
-  # After configured, whenever a "sql.active_record" notification is published,
-  # it will properly dispatch the event (ActiveSupport::Notifications::Event) to
-  # the sql method.
+  # After configured, whenever a <tt>"sql.active_record"</tt> notification is published,
+  # it will properly dispatch the event
+  # (<tt>ActiveSupport::Notifications::Event</tt>) to the sql method.
+  #
+  # Being an <tt>ActiveSupport::Notifications</tt> consumer,
+  # <tt>ActiveRecord::LogSubscriber</tt> exposes simple interface to check if
```

Comment:
<tt>ActiveRecord::LogSubscriber</tt> exposes _a_ simple interface

Reply:
Thx @gmcgibbon, this is now fixed -- it's always good to have some native speaker's feedback in PRs like this one. Anyway, this also made me realize that we should refer to `ActiveSupport::LogSubscriber` rather than `ActiveRecord::LogSubscriber`